src/components/molecules/OcTable/OcTable.vue

```diff
@@ -1,82 +1,228 @@
 <template>
-  <table v-bind="extractTableProps()">
-    <oc-thead v-if="hasHeader">
-      <oc-tr class="oc-table-header-row">
-        <oc-th
-          v-for="(field, index) in fields"
-          :key="`oc-thead-${field.name}`"
-          v-bind="extractThProps(field, index)"
-          @click.native="$emit(constants.EVENT_THEAD_CLICKED, field)"
+  <div>
+    <div
+      v-if="groupingSettings && groupingAllowed && groupingSettings.showGroupingOptions"
+      style="padding-bottom: 20px"
```

Comment:
Please use padding helper classes as documented in https://owncloud.design/#/Utilities/Padding